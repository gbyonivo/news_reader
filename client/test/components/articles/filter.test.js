import React from 'react';
import renderer from 'react-test-renderer';
import Filter from '../../../src/components/articles/filter';

describe('Filter component', () => {
  it('renders correctly', () => {
    const actual = renderer.create(
      <Filter defaultValue={'ter'} onChange={() => {}}/>
    );
    expect(actual).toMatchSnapshot();
  })
})