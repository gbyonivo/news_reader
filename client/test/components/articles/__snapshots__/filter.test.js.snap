// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filter component renders correctly 1`] = `
<input
  className="articlesFilter"
  defaultValue="ter"
  onChange={[Function]}
  placeholder="Search Articles"
  type="text"
/>
`;
